### ISSUE TITLE
applications on other monitors freezing up

### ISSUE THREAD

**Issue by windos98** [Codeusa/Borderless-Gaming#377]
i use voicemeeter, discord and obs, and while using borderless gaming, these programs start to lag. especially in OBS, where the frame rate of the capture actually drops. this is a huge issue as i wont be able to stream using your program.
as for voicemeeter, the audio graphs dont show up when im speaking and muting certain things with hot keys may not even work or happen very delayed.

in obs, you just cant see yours or others icons light up when speaking.

basically all applications running alongside whatever game you're playing will lag out
please fix

**Comment by windos98** [comment]
pretty disappointed that no one has responded or addressed this problem...
